tex/gsp-default.tex

```diff
@@ -47,7 +49,7 @@
 
 %% These macro enable the tuning of linepenalty, tolerance, pretolerance
 %% and emergencystretch
-\def\grelooseness{\looseness}%
+\def\grelooseness{-1}%
```

Comment:
Since all the penalties currently follow this model, it would probably make sense to implement a system similar to what I did for the distances that would allow changing all of them.